kubernetes/client/apis/core_v1_api.py

```diff
@@ -14792,8 +14793,8 @@ def patch_namespaced_config_map_with_http_info(self, name, namespace, body, **kw
             select_header_accept(['application/json', 'application/yaml', 'application/vnd.kubernetes.protobuf'])
 
         # HTTP header `Content-Type`
-        header_params['Content-Type'] = self.api_client.\
-            select_header_content_type(['application/json-patch+json', 'application/merge-patch+json', 'application/strategic-merge-patch+json'])
+        header_params['Content-Type'] = params.get('content_type', self.api_client.\
+            select_header_content_type(['application/json-patch+json', 'application/merge-patch+json', 'application/strategic-merge-patch+json'], 'PATCH', body_params))
```

Comment:
Kubernetes recently added a new type `application/apply-patch+yaml`. However it's likely hard to auto-detect the yaml type based on the body